### PR TITLE
Add Imagetools create command

### PR DESCRIPTION
**Pull request by darpham**
### Background
- What problem were you trying to solve
  - Would like to the ability to run `docker buildx imagetools create ...`
- Related links:
  - https://docs.docker.com/engine/reference/commandline/buildx_imagetools_create/

### Changes
- Add `create` method to `ImagetoolsCLI`
- pre-commit changes
  - `https://gitlab.com/pycqa/flake8`
    - change to `github` url as it's more popular (gitlab requires login)
  - Upgraded `isort`: https://github.com/PyCQA/isort/issues/2077
  - Use new repository names
    - `isort`
    - `ambv`

### Open questions / follow-up work
- Create unittest?

### CODE REVIEW

**Comment by gabrieldemarmiesse** [comment]
Hi! Thank you gor the pull request and the lint upgrade! Could yyou add some unit tests too? Thanks!

**Comment by darpham** [comment]
> Hi! Thank you gor the pull request and the lint upgrade! Could yyou add some unit tests too? Thanks!

Hello, I'm really dumb and have never written tests before 🥲 
I'm just really good at reading code and copy/pasting.

If you're willing to create tests, that'd be awesome.
Additionally, I'd be more than willing to hop on a zoom call to work together on this, so I can do it myself in the future. I love this repo, and would like to contribute more in the future as Docker continues to grow.

**Comment by gabrieldemarmiesse** [comment]
Apologies, i was quite busy those last few weeks, I skipped your message :(

No worries about it. While I won't really have the time to do a zoom call, I can write the test myself and you will be able to see what the test code should look like. Hopefully, that will help you understand a bit huw unit testing works.

I should have some time to make the unit tests Thursday :)

**Review by gabrieldemarmiesse — APPROVED** [review]

File: python_on_whales/components/buildx/imagetools/cli_wrapper.py [hunk 1]
```diff
@@ -10,3 +13,55 @@ def inspect(self, name: str) -> Manifest:
...
+        Create a new manifest list based on source manifests.
+        The source manifests can be manifest lists or single platform distribution manifests and
+        must already exist in the registry where the new manifest is created.
+        If only one source is specified, create performs a carbon copy.
+
+        The CLI docs is [here](https://docs.docker.com/engine/reference/commandline/buildx_imagetools_create/)
+        and it contains a lot more information.
+
+        # Arguments
+            source: The source manifest to create, change
```
Comment (gabrieldemarmiesse) [inline python_on_whales/components/buildx/imagetools/cli_wrapper.py:36]:
```suggestion
            sources: The sources manifest to create, change
```

**Pull request merged**